packages/client/tsconfig.json

```diff
@@ -28,7 +28,9 @@
     // "rootDir": "./",                                  /* Specify the root folder within your source files. */
     "moduleResolution": "node", /* Specify how TypeScript looks up a file from a given module specifier. */
     // "baseUrl": "./",                                  /* Specify the base directory to resolve non-relative module names. */
-    // "paths": {},                                      /* Specify a set of entries that re-map imports to additional lookup locations. */
+    "paths": {
+      "@openfeature/core": [ "../shared/src" ]
+    },                                                   /* Specify a set of entries that re-map imports to additional lookup locations. */
```

Comment:
this forces resolution of src/ instead of dist/ during development. No prod impact.